Cargo.toml

```diff
@@ -1,22 +1,16 @@
 [package]
-name = "firefly-iii"
-version = "0.2.4-alpha.0"
+name = "ff3"
+version = "1.5.5"
 authors = ["OpenAPI Generator team and contributors"]
 edition = "2018"
-description = "Rust API client for firefly-iii"
-license = "AGPL-3.0"
-documentation = "https://docs.rs/firefly-iii"
-repository = "https://github.com/barskern/firefly-iii-rs"
-readme = "README.md"
-keywords = ["openapi", "economy", "client"]
-categories = ["api-bindings"]
```

Comment:
These changes (above this comment in the Cargo.toml file) should be reverted and I will manually update the `version` when I release a new version to crates.io. The version of the crate cannot be related to the version of the API sadly (because we might want to alter the crate on the save version of the API).